dgr/common/dgr-manifest.go

```diff
@@ -74,6 +74,8 @@ type BuilderDefinition struct {
 
 type BuildDefinition struct {
 	MountPoints []MountInfo `json:"mountPoints,omitempty" yaml:"mountPoints,omitempty"`
+	Exclude     []string    `json:"exclude,omitempty" yaml:"exclude,omitempty"`
```

Comment:
I don't get why we need them, the builder scripts goal is to prepare the filesystem to be ready for tar. it should be done there